deepfence_agent/plugins/Makefile

```diff
@@ -0,0 +1,24 @@
+TARGET=bin/open_tracer
+
+all: $(TARGET)
+	-mkdir $(PWD)/bin
+	cp $(PWD)/open-tracer/target/release/echo bin/open_tracer
+
+bin/open_tracer: check-and-reinit-submodules-open-tracer
+	(cd open-tracer && cargo xtask build-ebpf --release)
```

Comment:
This Makefile is run inside the deepfence-agent-build container and hence, I assume we have the necessary step to install the cargo binaries ?

Reply:
https://github.com/noboruma/ThreatMapper/tree/open_tracer/deepfence_agent/build

Reply:
@shyam-dev Currently it's run on host, not inside the build container https://github.com/noboruma/ThreatMapper/blob/open_tracer/deepfence_agent/build.sh#L8-L15.

@noboruma Can you move that build process [here](https://github.com/noboruma/ThreatMapper/tree/open_tracer/deepfence_agent/build) so that users don't have to install any dependencies in the vm?
You might need to move [these lines](https://github.com/noboruma/ThreatMapper/blob/open_tracer/deepfence_agent/build.sh#L29-L42) before `echo "Building Scope"`.

Reply:
Let me do that - any reason for having scope built on the host directly? 
If not I will also move scope build process inside the build container.

Reply:
I have revamped the build process to use the build image:
https://github.com/deepfence/ThreatMapper/pull/181/commits/7c5dc749697c98579dfe054a0dd64621bb428b18
Plugins can also be generated locally, which might be helpful for development/debugging later on.